### ISSUE TITLE
Error: No type for DocBlock

### ISSUE THREAD

**Issue by trollsyn** [holidaypirates/nucleus#4]
Hi,

I have problems processing any files except `@color` nuclids (one-liners).

Windows 10 x64
Node 4.5.0
NPM 3.10.6

When I run `nucleus` I get this error message:

``` scss
Take care! No type for DocBlock in src/atoms/buttons.scss on line 23
 You need to specify at least the type of the element. Possible types are
 @nuclide (with @color and @mixin as special types), @atom (with @icon as
 special type), @molecule and @structure.

Skipping unknown entity type.
```

The content of the file `buttons.scss` is this:

``` scss
/**
 * Generic button styles for a subtile button on light background and a more present button
 * for darker / colorful backgrounds.
 *
 * @atom Button
 * @markup
 *  <button class="SG-button">.SG-button</button>
 *  <button class="SG-button SG-button--large">.SG-button</button>
 *  <div style="padding: 5px; background: #f15f79; margin: 1em -5px;">
 *    <button class="SG-button SG-button--white">.SG-button</button>
 *    <button class="SG-button SG-button--white SG-button--large">.SG-button</button>
 *  </div>
 */
.SG-button {
  color: $color--gray;
  background: none;
  border: 1px solid $color--light-gray;
  border-radius: 5px;
  padding: 10px 14px;
  display: inline-block;
  cursor: pointer;
  text-decoration: none;

  &:hover {
    border-color: $color--accent;
    color: $color--accent;
  }

  // Larger button
  &.SG-button--large {
    font-size: 16px;
    padding: 16px 36px;
  }

  // White button style
  &.SG-button--white {
    color: white;
    border-color: white;

    &:hover {
      background: white;
      color: $color--accent;
    }
  }
}
```

However, if I run any color declaration on one line, like this:

``` scss
/** @color White */
$color-white: #fff;
```

...it compiles without errors.

I tried guessing what could be wrong in my syntax, but without luck. Any clue?

**Comment by michaelseibt** [comment]
Hi trollsyn!

Thanks for your report. I've checked and validated that this seems to be a bug on Windows. Scanning your file works fine on Mac. I'll start investigating. 

Will come back to you with any news.

**Comment by trollsyn** [comment]
Ok, thank you for the quick reply!

I tried using Unix-EOL in Sublime Text 3 (`View > Line Endings > Unix`), and it does the trick as a workaround in case anyone else encounters the same problem.

Also great news that it'll be fixed in the future, thanks for making this :+1:

**Comment by joedrummer8993** [comment]
I am still having this problem as well.

**Comment by michaelseibt** [comment]
Hi @joedrummer8993,

which version of of Nucleus are you using? The bug should have been fixed from 1.0.5, so make sure you are using the latest version.

**Comment by joedrummer8993** [comment]
My mistake, I was on 1.0.4.  Thank you!
